features/other_a/reading.feature

```diff
@@ -245,3 +245,24 @@ Feature: Reading count
     And I go to the homepage
   Then I should see "Some Work V2"
     And I should not see "Some Work V1"
+
+  Scenario: A user cannot see hidden by admin works in their reading history
+
+  Given I am logged in as "writer"
+  When I post the work "Testy"
+  Then I should see "Work was successfully posted"
+  When I am logged out
+    And I am logged in as "reader"
```

Comment:
The login step includes https://github.com/otwcode/otwarchive/blob/1481bfda90156f36f776677cc059a23c21aa2e58/features/step_definitions/user_steps.rb#L136 so I think this can be simplified to
```suggestion
  When I am logged in as "reader"
```
(and similarly for the steps below)

Reply:
Fixed, I removed `I am logged out` from all tests.

I copied from some old tests in this same file that were using that.